packages/storybook/stories/components/gallery.mdx

```diff
@@ -0,0 +1,317 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import '../../assets/css/gallery.css';
+
+import { Overview as Accordion } from './accordion/accordion.stories';
+import { Overview as Badge } from './badge/badge.stories';
+import { Overview as Breadcrumb } from './breadcrumb/breadcrumb.stories';
+import { Overview as Button } from './button/button.stories';
+import { Overview as Card } from './card/card.stories';
+import { Overview as Checkbox } from './checkbox/checkbox.stories';
+import { Overview as Clipboard } from './clipboard/clipboard.stories';
+import { Overview as Code } from './code/code.stories';
+import { Overview as Datepicker } from './datepicker/datepicker.stories';
+import { Overview as Divider } from './divider/divider.stories';
+import { Overview as FileUpload } from './file-upload/file-upload.stories';
+import { Overview as FormField } from './form-field/form-field.stories';
+import { Overview as Icon } from './icon/icon.stories';
+import { Overview as Input } from './input/input.stories';
+import { Overview as Link } from './link/link.stories';
+import { Overview as Medium } from './medium/medium.stories';
+import { Overview as Message } from './message/message.stories';
+import { Overview as Modal } from './modal/modal.stories';
+import { Overview as Pagination } from './pagination/pagination.stories';
+import { Overview as Password } from './password/password.stories';
+import { Overview as PhoneNumber } from './phone-number/phone-number.stories';
+import { Overview as Popover } from './popover/popover.stories';
+import { Overview as ProgressBar } from './progress-bar/progress-bar.stories';
+import { Overview as Quantity } from './quantity/quantity.stories';
+import { Overview as Radio } from './radio/radio.stories';
+import { Overview as Range } from './range/range.stories';
+import { Overview as Select } from './select/select.stories';
+import { Overview as Skeleton } from './skeleton/skeleton.stories';
+import { Overview as Spinner } from './spinner/spinner.stories';
+import { Overview as Switch } from './switch/switch.stories';
+import { Overview as Table } from './table/table.stories';
+import { Overview as Tabs } from './tabs/tabs.stories';
+import { Overview as Tag } from './tag/tag.stories';
+import { Overview as Text } from './text/text.stories';
+import { Overview as Textarea } from './textarea/textarea.stories';
+import { Overview as Timepicker } from './timepicker/timepicker.stories';
+import { Overview as Toggle } from './toggle/toggle.stories';
+import { Overview as Tooltip } from './tooltip/tooltip.stories';
+
+<Meta title="ODS Components/Gallery" />
+
+<div id="gallery">
+
+  # UI Components
+
+  <div class="gallery__items">
+    <ods-card class="gallery__items__item"
+              color="neutral">
+      <Canvas of={ Accordion } sourceState='none' />
```

Comment:
try to always use the same quote everywhere.
For HTML attribute, we use `"...."`
(check some of the Overview you've added, as they expose those as example code, like in ods-text story for example)